### PR TITLE
Minor bugfixes.

### PR DESCRIPTION
**Pull request by jasomobica**
And a readme file.
Cleaned up minor bugs to have a clean closure compiler output.
Made the testPassed() and testFailed() argument optional.
Fixed run-closure.sh script

Signed-off-by: Janusz Sobczak <janusz.sobczak@mobica.com>

### CODE REVIEW

File: sdk/tests/deqp/framework/referencerenderer/rrRasterizer.js [hunk 1]
```diff
@@ -1,186 +0,0 @@
-/*-------------------------------------------------------------------------
```
Comment (kenrussell) [inline sdk/tests/deqp/framework/referencerenderer/rrRasterizer.js]: Is the removal of this file intentional?

**Comment by kenrussell** [comment]
Thank you Janusz. This looks good overall. One question per above.

File: sdk/tests/deqp/framework/referencerenderer/rrRasterizer.js (hunk 1 above)
Comment (jasomobica) [inline sdk/tests/deqp/framework/referencerenderer/rrRasterizer.js]: Yes. we're not using it.

**Comment by kenrussell** [comment]
Excellent. Thank you again for these important cleanups.

**Pull request merged**